dapps/onboarding.vue/src/i18n/de.json

```diff
@@ -251,11 +255,11 @@
       "accept-contact": "Kontaktanfrage akzeptieren",
       "do-not-accept-contact": "Weiter ohne Kontaktanfrage zu akzeptieren",
       "go-to-evan": "Weiter zum evan.network",
-      "invited": "Herzlich willkommen im evan.network <b>{alias} ({accountId})</b>.<br><br>Sie wurden von <b>{inviteeAlias}</b> in das evan.network eingeladen. Wenn Sie diesen Kontakt zu Ihrem Adressbuch hinzufügen, erhalten Sie <b>{eveAmount} EVE</b> als Einladungsgeschenk.",
+      "invited": "Herzlich willkommen im evan.network <b>{alias}</b>.<br><br>Sie wurden von <b>{inviteeAlias}</b> in das evan.network eingeladen. Wenn Sie diesen Kontakt zu Ihrem Adressbuch hinzufügen, erhalten Sie <b>{eveAmount} EVE</b> als Einladungsgeschenk.",
```

Comment:
maybe we should remove the part with the received EVE, since in the new UI int's not possible any more to attach EVEs on invites.